### PR TITLE
Tn-4460: Add dataModelMappings option to import packages command

### PR DESCRIPTION
**Pull request by fisgeci**
#### Description

Added dataModelMappings assignment to package variables when importing packages and their dependencies.

#### Checklist

- [x] I have self-reviewed this PR
- [x] I have tested the change and proved that it works in different scenarios
- [x] I have updated docs if needed

### CODE REVIEW

**Review by frkntrcnn — DISMISSED** [review]
lgtm

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
         -   [Pull package for EMS Store](#pull-package-for-ems-store)
     -   [Pull/Push individual assets from/to Studio](#pullpush-individual-assets-fromto-studio)
     -   [Overwrite Package In Studio](#overwrite-package-in-studio)
+    -   [Export multiple packages at once from Studio](#export-multiple-packages-at-once-from-studio)
```
Comment (promeris) [inline DOCUMENTATION.md:12]: I think a more correct term would be batch export and import. Let's replace it everywhere in the docs.

File: src/util/semantic-versioning.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export class SemanticVersioning {
+    private version: string;
+
+    constructor(version: string) {
+        this.version = version;
+    }
```
Comment (promeris) [inline src/util/semantic-versioning.ts:6]: nit: Can be simplified to the following.
```suggestion
    constructor(private version: string) {
    }
```

Comment (fisgeci) [reply src/util/semantic-versioning.ts:6]: Unfortunately this doesn't work. I tried it out as i wasn't sure but it throws a duplicate identifier compile error.

**Review by promeris — DISMISSED** [review]
Looks good. If you are done with testing let's also bump the patch version.

**Review by promeris — APPROVED** [review]

**Pull request merged**